### PR TITLE
Tracks to napari function in HDF5FileHandler

### PR DESCRIPTION
**Pull request by KristinaUlicna**
- [x] Fixed the coords axes order in `utils` by adding an optional argument `coords_axes` if needed. 
- [x] Added the napari tracks export fn to the `HDF5FileHandler` as an attribute.

### CODE REVIEW

**Comment by KristinaUlicna** [comment]
@quantumjot ready for review! 👍

File: btrack/dataio.py [hunk 1]
```diff
@@ -771,6 +771,13 @@ def lbep(self):
         logger.info(f'Loading LBEP/{self.object_type}')
         return self._hdf['tracks'][self.obj_type]['LBEPR'][:]
 
+    @h5check_property_exists('tracks')
+    def convert_tracks_to_napari(self, **kwargs):
+        """Return the 'data, properties, graph' to load to napari."""
+        data, properties, graph = utils.tracks_to_napari(self.tracks, **kwargs)
+        self.napari_tracks = (data, properties, graph)
```
Comment (paddyroddy) [inline btrack/dataio.py:778]: Can't see any real benefit of having this has two steps
```suggestion
        self.napari_tracks = utils.tracks_to_napari(self.tracks, **kwargs)
```

File: btrack/dataio.py [hunk 2]
```diff
@@ -771,6 +771,13 @@ def lbep(self):
...
+        return self.napari_tracks
```
Comment (paddyroddy) [inline btrack/dataio.py:779]: Does this need to return? Doesn't really matter either way

File: btrack/utils.py [hunk 3]
```diff
@@ -344,7 +344,12 @@ def _cat_tracks_as_dict(tracks: list, properties: list):
     return data
 
 
-def tracks_to_napari(tracks: list, ndim: int = 3, replace_nan: bool = True):
+def tracks_to_napari(
+    tracks: list,
+    ndim: int = 3,
+    coords_axes: list = None,
```
Comment (paddyroddy) [inline btrack/utils.py:350]: `Optional` comes from `typing`. Or could this be an empty list `[]` instead?

```suggestion
    coords_axes: Optional[list] = None,
```

File: btrack/utils.py [hunk 4]
```diff
@@ -353,11 +358,13 @@ def tracks_to_napari(tracks: list, ndim: int = 3, replace_nan: bool = True):
         A list of tracklet objects from BayesianTracker.
     ndim : int
         The number of spatial dimensions of the data. Must be 2 or 3.
+    coords_axes: list
+        The order of axes in the track objects. Defaults to ["z", "y", "x"].
```
Comment (paddyroddy) [inline btrack/utils.py:362]: To me it looks like it defaults to `None`?

**Review by paddyroddy — APPROVED** [review]
A few comments

**Comment by paddyroddy** [comment]
I've fixed the `pre-commit` checks

File: btrack/utils.py (hunk 3 above)
Comment (quantumjot) [inline btrack/utils.py:350]: How about this?
```suggestion
    coords_axes: List[str] = ["z", "y", "x"],
```

File: btrack/utils.py [hunk 5]
```diff
@@ -375,7 +383,10 @@ def tracks_to_napari(tracks: list, ndim: int = 3, replace_nan: bool = True):
     """
     # TODO: arl guess the dimensionality from the data
     assert ndim in (2, 3)
-    t_header = ["ID", "t"] + ["z", "y", "x"][-ndim:]
+    coords_axes = ["z", "y", "x"] if coords_axes is None else coords_axes
```
Comment (quantumjot) [inline btrack/utils.py:386]: Then you could get rid of this line and just make sure that:
```python
if not all([axis in ("x", "y", "z") for axis in coords_axes]):
  raise ValueError
```
just to check that these are only spatial dims?

File: btrack/dataio.py (hunk 2 above)
Comment (quantumjot) [reply btrack/dataio.py:787]: Do you need to store this as a member variable? I presume you just want to return it?

**Comment by quantumjot** [comment]
Am closing this as it is outdated.

**Pull request closed without merging**